definitions/pipelines/somatic_exome.cwl

```diff
@@ -119,13 +119,25 @@ inputs:
         label: "target_intervals: interval_list file of targets used in the sequencing experiment"
         doc: |
           target_intervals is an interval_list corresponding to the targets for the sequencing reagent.
-          These are essentially coordinates for regions you wanted to design probes for in the reagent.
+          These are essentially coordinates for regions designed probes for in the reagent.
           Bed files with this information can be converted to interval_lists with Picards BedToIntervalList.
           In general for a WES exome reagent bait_intervals and target_intervals are the same.
+    target_interval_padding:
+        type: int?
```

Comment:
I would not mark this as optional, since the tool doing the padding requires a value.  Even when required, the default will be used if a value is not supplied.